Bottom: 28aba4363731813becbee4e3b5f29f29546636c1
Top:    19c2f3b968a25bc73a607563b2b20d761eaee505
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-28 03:08:29 +0100

Refresh of use-trackball-to-allow-rotatio

---

diff --git a/src/Makefile.am b/src/Makefile.am
index 6333f45..6a0c335 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -161,7 +161,9 @@ LIBGTK_GDK_SRCS= \
 	hid/gtk/gtkhid-gdk.c
 
 LIBGTK_GL_SRCS= \
-	hid/gtk/gtkhid-gl.c
+	hid/gtk/gtkhid-gl.c \
+	hid/gtk/gui-trackball.c \
+	hid/gtk/gui-trackball.h
 
 GL_SRCS= \
 	hid/common/hidgl.c \
@@ -277,8 +279,6 @@ LIBGTK_SRCS = \
 	hid/gtk/gui-pinout-preview.h \
 	hid/gtk/gui-pinout-window.c \
 	hid/gtk/gui-top-window.c \
-	hid/gtk/gui-trackball.c \
-	hid/gtk/gui-trackball.h \
 	hid/gtk/gui-utils.c
 libgtk_a_SOURCES = ${LIBGTK_SRCS} hid/gtk/gtk_lists.h
